### PR TITLE
Update Clockwork_Pod.lua

### PR DESCRIPTION
**Pull request by xYuunaX**
Fix Mind Hoarder's PH

### CODE REVIEW

File: scripts/zones/FeiYin/mobs/Clockwork_Pod.lua [hunk 1]
```diff
@@ -17,5 +17,5 @@ function onMobDeath(mob, player, isKiller)
 end
 
 function onMobDespawn(mob)
-    dsp.mob.phOnDespawn(mob, ID.mob.DONNERGUGI_PH, 10, 3600) -- 1 hour
+    dsp.mob.phOnDespawn(mob, ID.mob.MIND_HOARDER_PH, 5400, 32400) -- 1.5 - 9 hours
```
Comment (KnowOne134) [inline scripts/zones/FeiYin/mobs/Clockwork_Pod.lua]: ..._PH, 10, (math.random(5400, 32400)) -- 1.5 to 9 hours

Comment (xYuunaX) [reply scripts/zones/FeiYin/mobs/Clockwork_Pod.lua]: thanks

File: scripts/zones/FeiYin/mobs/Clockwork_Pod.lua [hunk 2]
```diff
@@ -17,5 +17,5 @@ function onMobDeath(mob, player, isKiller)
 end
 
 function onMobDespawn(mob)
-    dsp.mob.phOnDespawn(mob, ID.mob.DONNERGUGI_PH, 10, 3600) -- 1 hour
+    dsp.mob.phOnDespawn(mob, ID.mob.MIND_HOARDER_PH, 10, (math.random(5400, 32400)) -- 1.5 to 9 hours
```
Comment (KnowOne134) [inline scripts/zones/FeiYin/mobs/Clockwork_Pod.lua]: Missing 1 more ) at end to close function

File: scripts/zones/FeiYin/mobs/Clockwork_Pod.lua [hunk 3]
```diff
@@ -17,5 +17,5 @@ function onMobDeath(mob, player, isKiller)
 end
 
 function onMobDespawn(mob)
-    dsp.mob.phOnDespawn(mob, ID.mob.DONNERGUGI_PH, 10, 3600) -- 1 hour
+    dsp.mob.phOnDespawn(mob, ID.mob.MIND_HOARDER_PH, 10, (math.random(5400, 32400))) -- 1.5 to 9 hours
```
Comment (TeoTwawki) [inline scripts/zones/FeiYin/mobs/Clockwork_Pod.lua]:
```lua
    dsp.mob.phOnDespawn(mob, ID.mob.MIND_HOARDER_PH, 10, math.random(5400, 32400)) -- 1.5 to 9 hours
```
Don't need extra parentheses

File: scripts/zones/FeiYin/mobs/Clockwork_Pod.lua (hunk 2 above)
Comment (TeoTwawki) [reply scripts/zones/FeiYin/mobs/Clockwork_Pod.lua]: incorrect, he had one to many

File: scripts/zones/FeiYin/mobs/Clockwork_Pod.lua (hunk 3 above)
Comment (KnowOne134) [reply scripts/zones/FeiYin/mobs/Clockwork_Pod.lua]: Need 3 at the end to close whole function. Travis already errored on it with 2 at end

Comment (TeoTwawki) [reply scripts/zones/FeiYin/mobs/Clockwork_Pod.lua]: because there was an extra leading one in front of the math random.

Comment (TeoTwawki) [reply scripts/zones/FeiYin/mobs/Clockwork_Pod.lua]: @KnowOne134 I'll make thsi really simple for you to visualize:

thing(stuff inside it)

thing(thing(stuff inside it))

thing(thing(thing(stuff inside it)))

thing(thing(stuff inside (it)))

(it) was wrong.

**Comment by TeoTwawki** [comment]
@xYuunaX In the future, please ensure your commit messages are descriptive of the changes being made, rather than just saying _"update filename"_

Using the github app or the website for your edits is notorious for encouraging that, and is a large part of why we steer people away from using those. We encourage command line [git for windows](https://gitforwindows.org/), or [tortoisegit](https://tortoisegit.org/), because they force a real commit message on the user.

**Comment by xYuunaX** [comment]
ok, will do.

Thank you

**Pull request merged**